docs/azure.rst

```diff
@@ -276,6 +313,7 @@ azure.batch.pools.<name>.vmType                 Specify the virtual machine type
 azure.batch.pools.<name>.vmCount                Specify the number of virtual machines provisioned by the pool identified with ``<name>``.
 azure.batch.pools.<name>.maxVmCount             Specify the max of virtual machine when using auto scale option.
 azure.batch.pools.<name>.autoScale              Enable autoscaling feature for the pool identified with ``<name>``.
+azure.batch.pools.<name>.fileShareRootPath      If mounting File Shares, this is the internal root mounting point. Must be ``/mnt/resource/batch/tasks/fsmounts`` for CentOS nodes or ``/mnt/batch/tasks/fsmounts`` for Ubuntu nodes (default is for CentOS).
```

Comment:
think already asked, there's no way to infer this? maybe using  `uname` to get the distribution if there's no better  way?

Reply:
The problem is that we need this info when we submit the tasks, we are not yet on the batch node to get this information on the fly. I tend to think that most users will go with the default and never touch this setting (like the others related to the image reference - they are all connected and should be considered at a whole).

Reply:
I see, what if we default this with the `sku` config setting (once merged).

Reply:
I'm not sure to follow this. 

Do you want to assign the `fileShareRootPath` automatically based on the `sku` from https://github.com/nextflow-io/nextflow/pull/2360? It's doable, but I, for one, would never be sure that all the ubuntu images mount under the same X path, all the windows images mount under the same Y path, etc. Not to mention custom images created by Azure users.

Because of that, I'd still leave to the user the option to set `fileShareRootPath`. Then if it's not set, we could go with a map like:

/.*ubuntu.*/ -> '/mnt/batch/tasks/fsmounts'
/.*centos.*/ -> '/mnt/resource/batch/tasks/fsmounts'

What do you think?

Reply:
> Because of that, I'd still leave to the user the option to set fileShareRootPath. Then if it's not set, we could go with a map like:

exactly